website/www/site/content/en/documentation/sdks/yaml-inline-python.md

```diff
@@ -123,6 +123,8 @@ define a `beam.PTransform` inline, e.g.
     constructor: __constructor__
     kwargs:
       source: |
+        import apache_beam as beam
```

Comment:
Beam is explicitly and intentionally in the namespace for convenience: https://github.com/apache/beam/blob/release-2.41.0/sdks/python/apache_beam/utils/python_callable.py#L105 so imports are superfluous and shouldn't be added. (There aren't any other "hidden" imports we have to worry about.)

Reply:
Ah I wasn't sure where the import was added so thanks for the link. In that case, I went the other direction and removed an import statement from one of the examples